extensions/datetime-format-converter/package.json

```diff
@@ -8,15 +8,36 @@
   "contributors": [
     "yifengkuaijian",
     "pernielsentikaer",
-    "hemang_kinra"
+    "hemang_kinra",
+    "jose-elias-alvarez"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂